donate.html

```diff
@@ -125,24 +131,18 @@
 
   <!-- Page Header -->
 
-  <div class="donate-header text-center">
+  <div class="hero">
     <div class="container">
-        <h2>
-          <span style="position: relative">
-          Donating to
-          <div class="d-none d-md-block">
-            <div class="large-spark pink-spark" style="top: -30px; left: -70px;"></div>
-            <div class="med-spark purp-spark" style="top: 50px; left: -120px;"></div>
-            <div class="small-spark blue-spark" style="top: 190px; left: -50px;"></div>
-            <div class="large-spark purp-spark" style="top: -35px; right: -40px;"></div>
-            <div class="small-spark pink-spark" style="top: 210px; right: -20px;"></div>
-            <div class="med-spark blue-spark" style="top: 90px; right: -110px;"></div>
-          </div>
```

Comment:
i suppose this was copied over from the old version, but the new design doesn't have any sparkles so can delete